src/index.html

```diff
@@ -6,21 +6,51 @@
         <title></title>
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-
         <link rel="apple-touch-icon" href="apple-touch-icon.png">
         <!-- Place favicon.ico in the root directory -->
-
+        <link rel="stylesheet" href="octicons/octicons.css">
         <link rel="stylesheet" href="css/normalize.css">
         <link rel="stylesheet" href="css/main.css">
         <script src="js/vendor/modernizr-2.8.3.min.js"></script>
     </head>
     <body>
-        <!--[if lt IE 8]>
-            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-        <![endif]-->
+      <header>
+      </header>
+      <section class= "page-content">
+        <section class="user-sidebar">
+          <img src="https://avatars.githubusercontent.com/u/583231?v=3">
+          <section class="name-login">
+            <h3>The Octocat</h3>
+            <h4>octocat</h4>
+        </section>
+        <section class="details">
+
+          <p><span class="octicon octicon-organization"></span> GitHub</p>
+          <p><span class="octicon octicon-location"></span>  San Francisco</p>
+          <p><span class="octicon octicon-mail"></span>  octocat@github.com</p>
+          <p><span class="octicon octicon-link"></span>  http://www.github.com/blog</p>
+          <p><span class="octicon octicon-clock"></span>  Joined on Jan 25, 2011</p>
+        </section>
```

Comment:
Interesting. I bet getting the spacing around those `<p>` tags was interesting... This seems more like a "list" to me. Those `<span>` tags definitely work, but they're _so_ ugly (and by that I mean non-semantic). How does that `.octicon` thing work? Could you duplicate that with your own (S)CSS?